bokehjs/src/coffee/models/ranges/data_range1d.coffee

```diff
@@ -17,6 +17,8 @@ class DataRange1d extends DataRange.Model
       follow_interval: [ p.Number        ]
       default_span:    [ p.Number, 2     ]
       bounds:          [ p.Any           ] # TODO (bev)
+      min_interval: [ p.Any ]
+      max_interval: [ p.Any ]
```

Comment:
Given that bounds is `p.Any`, I think it's reasonable to have `p.Any` until we make a property that describes all range properties.